test/rest_api/test_storage_service.py

```diff
@@ -470,3 +470,12 @@ def test_storage_service_keyspace_upgrade_sstables(cql, this_dc, rest_api):
                 # non-existing table
                 resp = rest_api.send("GET", f"storage_service/keyspace_upgrade_sstables/{keyspace}", { "cf": f"{test_tables[0]},XXX" })
                 assert resp.status_code == requests.codes.bad_request
+
+def test_storage_service_system_keyspace_repair(rest_api):
+    resp = rest_api.send("POST", f"storage_service/repair_async/system")
+    resp.raise_for_status()
+    assert resp.json() > 0, "Repair got invalid sequence number"
+
+    resp = rest_api.send("GET", "task_manager/list_module_tasks/repair")
+    resp.raise_for_status()
+    assert not resp.json(), "Repair task for keyspace with local replication strategy was created"
```

Comment:
Hmm, can other (repair) tests run concurrenrly on the same node?  If not, than this LGTM.
Otherwise, how about getting the particular task status and validating its task id isn't found? (or making sure it isn't in the list returned above, if it happens to be non-empty)